src/auth/screens/events.screen.js

```diff
@@ -475,6 +485,17 @@ class Events extends Component {
     return item.id;
   };
 
+  changeTabBarVisibility = event => {
+    const scrollOffset = event.nativeEvent.contentOffset.y;
```

Comment:
Are we sure we will not get any undefined in this object? Maybe we can use `lodash.get` just to be sure :)
If you're 100% sure, disregard my comment